src/providers/simple/simple_access.c

```diff
@@ -165,6 +162,19 @@ int simple_access_obtain_filter_lists(struct simple_ctx *ctx)
               "No rules supplied for simple access provider. "
```

Comment:
I guess this check should be done after bunch of `talloc_steal()` below..?

Reply:
done